ts/features/itwallet/presentation/components/ItwClaimsSections.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { VSpacer } from "@pagopa/io-app-design-system";
+import { pipe, constNull } from "fp-ts/lib/function";
+import * as O from "fp-ts/lib/Option";
+import * as RA from "fp-ts/lib/ReadonlyArray";
+import I18n from "../../../../i18n";
+import { groupCredentialClaims } from "../../common/utils/itwClaimsUtils";
+import { StoredCredential } from "../../common/utils/itwTypesUtils";
+import { ItwCredentialClaimsSection } from "../../common/components/ItwCredentialClaimsSection";
+import { ItwCredentialClaim } from "../../common/components/ItwCredentialClaim";
+
+type Props = {
+  credential: StoredCredential;
+};
+
+/**
+ * Renders claims within sections defined by the hardcoded configuration in app.
+ */
+export const ItwClaimsSections = ({ credential }: Props) => {
+  const groupedClaims = groupCredentialClaims(credential);
+
+  return (
+    <>
+      {pipe(
+        groupedClaims.personalData,
+        O.fromNullable,
+        O.filter(RA.isNonEmpty),
+        O.fold(constNull, claims => (
+          <>
+            <VSpacer size={24} />
+            <ItwCredentialClaimsSection
+              title={I18n.t(
+                "features.itWallet.presentation.credentialDetails.personalDataTitle"
+              )}
+              canHideValues
+              claims={claims}
+            />
+          </>
+        ))
+      )}
+      {/* TODO: add section for mDL */}
```

Comment:
Is this TODO linked to a Jira story? What do you think to add a Jira ID if appropriate?